vulnerabilities/models.py

```diff
@@ -369,7 +367,27 @@ class VulnerabilitySeverity(models.Model):
         ),
     )
 
-    value = models.CharField(max_length=50, help_text="Example: 9.0, Important, High")
+    value = models.CharField(
+        max_length=50,
+        help_text="Score Value expressed as a number such as 9.0 that can be sorted or compared",
+    )
+
+    scoring_elements = models.CharField(
+        max_length=100,
+        null=True,
+        help_text="Supporting a scoring elements as a string "
+        "For example a CVSS vector Important, High, Medium ,Low."
+        "Typically used to compute the value",
+    )
+
+    def save(self, *args, **kwargs):
```

Comment:
IMHO we should not save here

Reply:
I think we should compute it in `severity_system.py`